missing-table.sh

```
@@ -0,0 +1,427 @@
+# Function to get process info by port
+get_process_by_port() {
+    lsof -ti :$1 2>/dev/null
+}
+
+# Function to get all missing-table related processes
+get_missing_table_processes() {
+    # Find processes related to missing-table (various ways Claude might start them)
+    local pids=""
+
+    # Method 1: By port
+    local backend_pid=$(get_process_by_port $BACKEND_PORT)
```

Shellcheck (0.11.0)

[warning] Declare and assign separately to avoid masking return values.

(SC2155)